detectors/python-mass-assignment/python-mass-assignment_compliant.py

```
@@ -0,0 +1,15 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: MIT-0
+
+# {fact rule=python-mass-assignment@v1.0 defects=0}
+
+from django.shortcuts import render
+
+def compliant(request):
+    # Compliant: The code explicitly specifies the fields to be assigned values from user input, preventing mass assignment vulnerabilities.
+    Whatzit.objects.create(
```

Ruff (0.13.1)

Undefined name `Whatzit`

(F821)